docs/en/connector-v2/sink/FtpFile.md

```diff
@@ -127,6 +128,10 @@ Please note that, If `is_enable_transaction` is `true`, we will auto add `${tran
 
 Only support `true` now.
 
+### batch_size [int]
+
+Write a file for every number of pieces of data
```

Comment:
We need let user know `batch_size` is the maximum number of rows in a file. For SeaTunnel Engine, The number of lines in the file is determined by `batch_size` and `checkpoint.interval` jointly decide. If the value of `checkpoint.interval` is large enough, sink writer will write rows in a file until the rows in the file large than `batch_size`. If `checkpoint.interval` is small, the sink writer will create a new file when a new checkpoint trigger.